shared/src/components/exercise/mode.cjsx

```diff
@@ -31,7 +31,7 @@ ExMode = React.createClass
 
   componentDidUpdate: (prevProps) ->
     {mode, focus} = prevProps
-    @setFreeResponseFocusState() if mode is 'free-response' and focus isnt @props.focus
+    @setFreeResponseFocusState() if mode is 'free-response'
```

Comment:
Does `setFreeResponseFocusState` cause scrolling?
The intent of the card is that the free reponse area get the focus when the question loads. This works in the `componentDidMount`, which is to say the first time a free-response field draws, but if you go through more questions and come to another one, it does not get focus.